discover/portal/articles/210-setting-up/02-instance-settings/02-instance-settings.markdown

```diff
@@ -0,0 +1,217 @@
+# Instance Settings [](id=instance-settings)
+
+Virtual instance settings are configured in the Instance Settings section of the
+Control Panel. 
+
+Instance Settings has four sections: *Configuration*, *Identification*,
+*Social*, and *Miscellaneous*. This article covers the first and largest
+section, Configuration. The remaining sections are covered in the next article.
+
+![Figure 1: The configuration options in Instance Settings are divided into four categories.](../../../images/instance-settings-navigation.png)
+
+The Configuration section of In stance Settings is divided into seven
+sub-sections. The first is General.
+
+## General [](id=general)
+
+The General section has three headings: Main Configuration, Navigation, and
+Additional Information. Under Main Configuration, set the name of the entity
+responsible for running the virtual instance. Additionally, define the name of
+your virtual instance's default site, set the mail domain, virtual host and
+content delivery network address.
+
+Under the Navigation heading, set a home page for your virtual instance as well
```

Comment:
I think it would be more clear if we use bullet points for this paragraph and describe each of the three URLs that can be configured.

Reply:
(1) and (2)   This has been like this since the beginning of times. It should probably be reviewed. In any case I'm pretty sure that changing it in one place also changes the other.
(3) You can assume readers know what a CDN is. As the description you can say: Enter one of more URLs pointing to a CDN configured to serve the static resources. .